src/js.rs

```diff
@@ -113,6 +114,8 @@ impl JsGenerator {
                 });
             }
 
+            const { ReadableStream } = require("node:stream/web");
```

Comment:
In the browser this is in the global scope, so this should probably be guarded.. something like
```suggestion
            const { ReadableStream } = (typeof window == "object" && { ReadableStream }) || require("node:stream/web");
```

Reply:
will that satisfy webpack? I started adapting the hack, but then thought that there's no point unless I actually test it, so I left it.